debian/changelog

```diff
@@ -0,0 +1,5 @@
+dangerzone (0.7.0) bionic; urgency=low
```

Comment:
I've committed the file produced by `stdeb` at first, but I'm not sure why "bionic" is mentioned here, and if we should keep this changelog as is.

Reply:
It's supposed to be what distribution you're targeting. For our purposes it doesn't really matter, because our repository software ignores this field (well, we tell it too). In SecureDrop Client we just write "unstable" and [have logic](https://github.com/freedomofpress/securedrop-client/blob/main/scripts/fixup-changelog.sh) to embed the distro name in the version. But I don't think you need that since y'all use one deb across all distro versions.

Reply:
Thanks, I've set it to "unstable" as well then. Yep for now it doesn't seem needed to generate different .deb files